src/Mod/PartDesign/Gui/Utils.h

```diff
@@ -23,7 +23,7 @@
 #ifndef UTILS_H_CS5LK2ZQ
 #define UTILS_H_CS5LK2ZQ
 
-/** \file Utils.h
+/** \file PartDesign/Gui/Utils.h
  *  This fiel contains some utility function used over PartDesignGui module
```

Comment:
Sorry, not sure why the github UI is not showing it more prominently but i was referring to line 27
https://github.com/FreeCAD/FreeCAD/pull/564#discussion-diff-103234538L27